pkg/pureutils/purefa_rest2x.go

```diff
@@ -42,6 +42,25 @@ func ListAllDestroyedVolumesFromFA(faClient *flasharray.Client) ([]flasharray.Vo
 	return volumes, nil
 }
 
+// IsVolumeExistsonFA checks if volume exists in FA
+func IsVolumeExistsonFA(faClient *flasharray.Client, volumeName string) (bool, error) {
+	params := make(map[string]string)
+	params["names"] = volumeName
+	params["destroyed"] = "false"
+	vols, err := faClient.Volumes.ListAllAvailableVolumes(params, nil)
```

Comment:
can we change the name of this function to `ListFAVolumes`? This was we won't be confused on the method name